src/app/digital/models/ioobjects/inputs/ConstantNumber.ts

```diff
@@ -0,0 +1,59 @@
+import {DigitalComponent} from "digital/models/DigitalComponent";
+import {ClampedValue} from "math/ClampedValue";
+import {DecimalToBCD} from "math/MathUtils";
+import {serializable, serialize} from "serialeazy";
+import {V} from "Vector";
```

Comment:
nit: order should be:
serialeazy on top
vector below that
math imports below that
digital/models import last